dicodile/data/gait.py

```diff
@@ -40,13 +50,96 @@ def get_gait_data(subject=1, trial=1):
     gait_dir.mkdir(parents=True, exist_ok=True)
     gait_zip = download(
         "http://dev.ipol.im/~truong/GaitData.zip",
-        gait_dir / "GaitData.zip"
+        gait_dir / "GaitData.zip",
+        verbose=verbose
```

Comment:
Mayeb put explicitely that here you use the fact that `replace=False` to avoid redownloading this multiple times.